.github/workflows/REUSABLE_backend.yml

```diff
@@ -21,6 +21,12 @@ on:
         required: false
         default: '.'
 
+      # Used with mono-repos, defaults to `backend_directory`.
+      composer_directory:
```

Comment:
Note: maybe `composer_root_directory`?  Should be clearer that this is where composer is run from, not where composer is located.

Reply:
It doesn't really matter since its removed in the follow up PR 👍🏼